### ISSUE TITLE
Cursor movement speed while swiping over spacebar

### ISSUE THREAD

**Issue by FrankT423** [futo-org/android-keyboard#754]
### Use Cases

When you are parsing through a long passage, you either have to click on the text itself, or you have to repetitively swipe over the space bar to get to where you'd want to go.

### Proposal

Having either an optional acceleration of parsing speed the faster you swipe, or a setting that changed the speed of the parsing (linearly).

You could also make it so if you lift your finger above the space bar it while swiping vertically it speeds it up (i've seen similar implementations in video parsing). the farther your finger is from the space bar (vertically) the faster it could go.

**Comment by haldi4803** [comment]
What you want is a smart skip that switches from single character to complete words after skipping through 3-5 Words.

**Comment by ftars2** [comment]
Yeah, something like that could also work. In general I'd say it feels a little too slow so a general speed option could be useful, but that would also work.

**Comment by FrankT423** [comment]
Yeah, something like that could also work. In general I'd say it feels a little too slow so a general speed option could be useful, but that would also work.
